native/java/org/jpype/manager/TypeManager.java

```diff
@@ -368,6 +369,10 @@ private ClassDescriptor createClass(Class<?> cls, boolean special)
       modifiers |= ModifierCode.THROWABLE.value;
     if (Serializable.class.isAssignableFrom(cls))
       modifiers |= ModifierCode.SERIALIZABLE.value;
+    if (Buffer.class.isAssignableFrom(cls))
+    {
+      modifiers |= ModifierCode.BUFFER.value | ModifierCode.SPECIAL.value;
+    }
```

Comment:
This is an edge case.  Most of the special wrappers get made at boot time, but this one did not have an order problem so we will use a lazy creation pattern.